spec/models/category_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'Category' do
+  let(:category) { build(:default_category) }
+
+  describe 'invalid category names' do
+    it 'can not be accepted' do
+      invalid_categorynames = [
+        'a' * (32 + 1),
+        ' ' * (32 + 1),
+        ' ' * 32,
+      ]
+      expect(category).not_to allow_value(*invalid_categorynames).for(:name_en)
+      expect(category).not_to allow_value(*invalid_categorynames).for(:name_ja)
+    end
+  end
+
+  describe 'appropriate name' do
+    it 'can be returned' do
+      category = create(:default_category)
+      expect(category.name).to eq("Category1")
+      I18n.locale = :ja
+      expect(category.name).to eq("Category1_ja")
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [Sider](https://sider.review) -->